packages/stream_video_flutter/lib/src/call_controls/controls/toggle_layout_option.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+import '../../../stream_video_flutter.dart';
+import '../../theme/stream_icons.dart';
```

Comment:
I think we should avoid relative imports and either import from package or rollup files.

Reply:
I don't like then also but they are used across whole SDK so it would require some other cleanup task

Reply:
Agreed, I can do that, it would be a fast win for me 🙂